src/test/java/com/wikia/webdriver/testcases/facebooktests/FacebookButtonTests.java

```diff
@@ -0,0 +1,79 @@
+package com.wikia.webdriver.testcases.facebooktests;
+
+import org.testng.annotations.Test;
```

Comment:
The convention I've seen elsewhere is to include this line directly after all other imports, like [this](https://github.com/Wikia/selenium-tests/blob/master/src/test/java/com/wikia/webdriver/testcases/venustests/TestTablesOnVenus.java#L7). We should probably do the same here to be consistent.